FordLabs/PeopleMover
# Accessibility focus state flow

## Issue
Resolves 81

## What was done
- [x] Meet accessibility standards for the top level of individual space pages

## How to test
1. Tab through a space page and ensure the order is logical
2. Ensure that a space page can be understood and navigated using the screen reader
